### DISCUSSION TITLE
Any plans to upgrade redis-py to 4.5.x?

### DISCUSSION THREAD

**Discussion by amithanamakkanavar** [Grokzen/redis-py-cluster#515] [General]
This library currently supports redis-py <4.0.0, but redis-py has already release 4.5.5 and they have fixed many vulnerabilities.
Wanted to know any plans on upgrading to 4.5.5 and above version os redis-py.

**Comment by Grokzen** [comment]
@amithanamakkanavar In general no, this repo has been archived. Your suggested migration plan is to move from this repo over to redis-py itself. The critical security flaw thingy was more thought of for the code in here over external dependencies. Also upgrading major versions to make them compatible can cause lots of major changes that would require lots of invested time as well
